onnxscript/rewriter/generic_pattern.py

```diff
@@ -88,6 +88,26 @@ def _to_match_result(pmr: PatternMatchResult) -> orp.MatchResult:
     return result
 
 
+def _value_to_str(value: ir.Value | orp.ValuePattern) -> str:
+    return value.name if value.name is not None else "anonymous:" + str(id(value))
+
+
+def _opt_value_to_str(value: ir.Value | orp.ValuePattern | None) -> str:
+    return _value_to_str(value) if value is not None else "None"
+
+
+def _node_to_str(node: ir.Node) -> str:
```

Comment:
```suggestion
def _node_to_short_str(node: ir.Node) -> str:
```

maybe something like this to distinguish from the native `__str__`?

Reply:
In a given context like this, it is (implicitly) the specialized representation desired by that context ... I think what you are suggesting would make sense if we try to share this, eg., move this to be a method into `ir.Node` as a `to_short_str()` method?

Reply:
Sounds reasonable, thanks!